introduction.md

```diff
@@ -1,3 +1,15 @@
 My name is Nicolas the Pretty Good, but you can call Nicolas the Pretty Good. I am a developer stationed in Miami, soon to move to San Francisco to dwelve deeper into the cyber tribe. I have a background in psychology, which leads me to often reflect on the similarities between the mind and programs, brains and computers. I love to create, solve problems, read, learn, humor in general. 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?